provider/common/bootstrap.go

```diff
@@ -304,9 +307,9 @@ func BootstrapInstance(
 		}
 		// This is the last zone in the list, error.
 		if len(zones) > 1 {
-			return nil, nil, nil, errors.Errorf(
-				"cannot start bootstrap instance in any availability zone (%s)",
-				strings.Join(zones, ", "),
+			return nil, nil, nil, fmt.Errorf(
+				"cannot start bootstrap instance in any availability zone (%s): %w",
```

Comment:
since zone errors might be long, maybe add a `\n` before the `%w`

Reply:
errors.Join does that. Still want me to do another one ?

Reply:
after `cannot start bootstrap instance in any availability zone (%s):`